### DISCUSSION TITLE
Optionally show homepage from CONTROL in vcpkg search

### DISCUSSION THREAD

**Discussion by rconde01** [microsoft/vcpkg#17906] [Ideas]
****

**Is your feature request related to a problem? Please describe.**
I wanted google benchmark. If I search "benchmark", I get "benchmark            1.5              A library to support the benchmarking of functions, similar to unit-tests."   It would be nice to know more details, like perhaps author, to understand it's really what i want.

**Proposed solution**
Show the homepage when running search if --homepage is specified.

**Describe alternatives you've considered**
Alternatively, if available, there could be a --details, --author etc. switch which provides better origin information.

**Comment by Rastaban** [comment]
We recently added the "Homepage" tag to the Control files with something like this as an end goal, we just haven't gotten around to designing and implementing the final step yet.  Would you be interested in implementing the final piece?  If you don't have time to do it then that is okay, we can make it up for grabs to anyone.

**Comment by rconde01** [comment]
I think I can get it done...what is the design you were expecting?

**Comment by Rastaban** [comment]
I have not put too much thought into the design.  As a starting point I like the --details switch you suggested.  It would probably work best in a multi-line format, that way it can also show long descriptions.

I have been busy with some new responsibilities so I am going to hand you off to @vicroms.  Good luck!
